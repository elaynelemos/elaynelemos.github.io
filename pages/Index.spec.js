import { createLocalVue, mount } from '@vue/test-utils'
import Index from '.'

const localVue = createLocalVue()

describe('Index', () => {
  test('properly mounted', () => {
    const wrapper = mount(Index, {
      localVue,
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
