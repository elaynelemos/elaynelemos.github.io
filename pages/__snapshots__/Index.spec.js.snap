// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index properly mounted 1`] = `
<section class="container">
  <div><img src="../assets/img/logo.jpg">
    <h2 class="subtitle">
      Website in progress! :D
    </h2>
    <div class="links"><a href="https://linkedin.com/in/elaynelemos" target="_blank" class="button--green">LinkedIn</a> <a href="https://github.com/elaynelemos" target="_blank" class="button--grey">GitHub</a></div>
    <div class="lds-ellipsis">
      <div></div>
      <div></div>
      <div></div>
      <div></div>
    </div>
  </div>
</section>
`;
